cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala

```diff
@@ -163,6 +163,31 @@ class DatabasesTest extends DocumentingTest with QueryStatisticsTestSupport {
         }
       }
     }
+    section("Altering databases", "administration-databases-alter-database", "enterprise-edition") {
+      p("Databases can be modified using the command `ALTER DATABASE`. In this Neo4j version, the only thing that can be modified is the database access. " +
+        "At creation, a database always has read-write access. " +
+        "The database access can be changed to read-only using the `ALTER DATABASE` command with the sub clause `SET ACCESS READ ONLY`. " +
+        "Thereafter, the database access can be changed back to read-write using the sub clause `SET ACCESS READ WRITE`. " +
+        "Altering the database access is allowed at all times, whether a database is online or offline. ")
+      p("Database access can also be managed using the configuration parameters `dbms.databases.default_to_read_only`, `dbms.databases.read_only`, and " +
+        "`dbms.database.writable`. For details, see <<operations-manual#manage-databases-parameters, Configuration parameters>>. " +
+        "If the database access set by the `ALTER DATABASE` command is read-only while the database according to those settings is read-write (or vice versa), " +
+        "the database will be read-only and prevent write queries.")
+      query("ALTER DATABASE customers SET ACCESS READ ONLY", ResultAssertions((r) => {
+        assertStats(r, systemUpdates = 1)
+      })) {
+        statsOnlyResultTable()
+      }
+      p("The database access can be seen in the `access` output column of the command `SHOW DATABASES`.")
+      query("SHOW DATABASES yield name, access", assertDatabasesShown) {
+        resultTable()
+      }
+      p("This command is optionally idempotent, with the default behavior to throw an exception if the database does not exists. " +
```

Comment:
```suggestion
      p("This command is optionally idempotent, with the default behavior to fail with an error if the database does not exist. " +
```
Maybe don't talk about exceptions, that's more relevant to code.

Reply:
I think that is a good point, will update it also for DROP which I stole this from